js/src/components/paid-ads/billing-card/billing-card.js

```diff
@@ -15,21 +16,33 @@ import fallbackBillingUrl from './fallbackBillingUrl';
 import { GOOGLE_ADS_BILLING_STATUS } from '.~/constants';
 import './billing-card.scss';
 
-const { APPROVED } = GOOGLE_ADS_BILLING_STATUS;
+const { APPROVED, PENDING } = GOOGLE_ADS_BILLING_STATUS;
 
 /**
  * Renders a success notice card or a setup card according to the billing status
  * of the connected Google Ads account.
  */
 export default function BillingCard() {
+	const [ showNotice, setShowNotice ] = useState( false );
 	const { billingStatus, hasFinishedResolution } =
 		useGoogleAdsAccountBillingStatus();
 
+	useEffect( () => {
+		// only show notice if we are in pending state before
+		if ( billingStatus.status === PENDING && ! showNotice ) {
+			setShowNotice( true );
+		}
+	}, [ billingStatus, showNotice ] );
+
 	if ( ! hasFinishedResolution ) {
 		return <SpinnerCard />;
 	}
 
 	if ( billingStatus.status === APPROVED ) {
+		if ( ! showNotice ) {
+			return null;
+		}
```

Comment:
This makes the appearance and disappearance of the last loading spinner even more inappropriate.

### Refresh the webpage at step 4 of the onboarding flow

https://github.com/user-attachments/assets/86113cc6-4c2b-4d4d-89b4-22de7436a4ab

### Enter the campaign creation page

https://github.com/user-attachments/assets/3f7fbb47-7339-434b-9750-cac8dd6f1f9e

Reply:
@eason9487 We are now showing the `BillingCard` component only during onboarding or the setup Ads flow.